data/mods/Magiclysm/Spells/attunements/Void_Mage.json

```
@@ -0,0 +1,164 @@
+    "damage_type": "pure",
+    "min_dot": 5,
+    "max_dot": 55,
+    "dot_increment": 1.25,
+    "max_level": 40,
+    "min_duration": 500,
+    "max_duration": 5000,
+    "duration_increment": 115,
+    "spell_class": "VOID_MAGE",
+    "base_casting_time": 200,
+    "energy_source": "MANA",
+    "base_energy_cost": 300,
+    "final_energy_cost": 150,
+    "energy_increment": -4,
+    "difficulty": 9,
+    "extra_effects": [ { "id": "voidmage_final_grasp_slowdown" } ]
+  },
+  {
+    "id": "voidmage_final_grasp_slowdown",
+    "type": "SPELL",
+    "name": "Final Grasp slowdown",
+    "description": "Removes some moves from an enemy.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "mod_moves",
+    "shape": "blast",
+    "max_level": 40,
+    "min_damage": -100,
+    "max_damage": -3000,
+    "damage_increment": -70
+  },
+  {
+    "type": "SPELL",
+    "id": "voidmage_horizon_vortex",
+    "name": "Horizon Vortex",
+    "description": "Creates a chaotic energy field, that shutter enemies inside it. Its pretty random, and can either kill everyone in area or produce almost nothing.",
+    "//": "how spell work: you spawn the virtual point (X), that start randomly spam random spells from `extra_effects` list, increasing level decrease casting time and mana cost, and damage of half extra effects (cause it will be totally awful if damage was at 100% random) - another half have random damage from 22 to 82, number of spell that X can cast also totally random. So using this spell surrounded by zombie crowd is like playing russian roulette - it can kill all mobs near you, but also can kill almost no one. Average it can be enough good in save situations",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SILENT", "SOMATIC", "WONDER", "NO_LEGS", "RANDOM_DAMAGE", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "min_damage": 5,
+    "max_damage": 35,
+    "damage_increment": 1,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 0.4,
+    "max_level": 40,
+    "spell_class": "VOID_MAGE",
+    "base_casting_time": 1200,
+    "final_casting_time": 600,
+    "casting_time_increment": -15,
+    "energy_source": "MANA",
+    "base_energy_cost": 1600,
+    "final_energy_cost": 1000,
+    "energy_increment": -15,
+    "difficulty": 10,
+    "extra_effects": [
+      { "id": "horizon_vortex_fire" },
+      { "id": "horizon_vortex_acid" },
+      { "id": "horizon_vortex_bash" },
+      { "id": "horizon_vortex_pure" },
+      { "id": "horizon_vortex_cold" },
+      { "id": "horizon_vortex_cut" },
+      { "id": "horizon_vortex_electric" },
+      { "id": "horizon_vortex_stab" }
+    ]
+  },
+  {
+    "abstract": "horizon_vortex_abstract",
+    "name": "Horizon Vortex abstract",
+    "type": "SPELL",
+    "description": "Give some random damage for Horizon Vortex spell.  Subspell; you aren't allowed to see it except in debug mode.",
```

```suggestion
    "description": "Causes a random amount of heat damage in the Horizon Vortex spell.  Subspell; you aren't allowed to see it except in debug mode.",
```

---

Its abstract without damage, so it doesnt have any damage at all, also type of damage is in spell name, so i think its not nesessary to duplicate it

---

```suggestion
    "description": "Causes a random amount of damage for the Horizon Vortex spell.  Subspell; you aren't allowed to see it except in debug mode.",
```
Actually, I'm not sure if you'd see it in debug mode - I hadn't noticed it was an abstract (newly possible!).

---

I see; this is for copying over then putting in the different names and types of damage.
```suggestion
    "description": "Causes a random amount of damage in the Horizon Vortex spell.  Subspell; you aren't allowed to see it except in debug mode.",
```